content/en/infrastructure/containers/orchestrator_explorer.md

```diff
@@ -140,6 +140,209 @@ All of these columns support sorting, which helps you to pinpoint individual wor
+
+>  ***Notes**: You might find the same key-value pairs as both a tag and label (or annotation) - this is dependent on how your cluster is configured.*
+
+The following resource attributes are supported in arbitrary **String Matching**:
+- `metadata.name`
+- `metadata.uid`
+- IP Addresses found in:
+  - Pods
+  - Nodes (Internal and External)
+  - Services (Cluster, External, and Load Balancer IPs)
+
+This means that you do not need to specify a key to search for a resource by name, or IP. Quotes are not required unless your string search includes certain special characters.
+
+#### Comparators
+
+All terms support the `:` equality operator. [Metric Value](#Resource-Utilization-Filters) terms support numeric comparisons as well:
+
+- `:>` Greater than (for example, `metric#cpu_usage_avg15:>0.9`)
+- `:>=` Greater than or equal
+- `:<` Less than
+- `:<=` Less than or equal
+
+#### Operators
+
+To combine multiple terms into a complex query, you can use any of the following case sensitive Boolean operators:
+
+| Operator | Description | Example |
+|---|---|---|
+| `AND` | **Intersection**: both terms are in the selected events (if nothing is added, AND is taken by default) | `a AND b`   |
+| `OR` | **Union**: either term is contained in the selected events                                             | `a OR b`   |
+| `NOT` / `-` | **Exclusion**: the following term is NOT in the event (apply to each individual raw text search) | `a AND NOT b` or<br>`a AND -b` |
+|  `( )` | **Grouping:** specify how to group terms logically. | `a AND (b OR c)` or<br>`(a AND b) or c` |
+
+##### `OR` Value Shorthand
+
+Multiple terms sharing the same key can be combined into a single term if they are all `OR`ed. For example, this query:
+
+```
+app_name:web-server OR app_name:database OR app_name:event-consumer
+```
+
+Can be reduced to:
+
+```
+app_name:(web-server OR database OR event-consumer)
+```
+
+### Wildcards
+
+You can use `*` wildcards as part of a term to filter by partial matches, both for values and keys! Some examples:
+
+- `kube_job:stats-*`: Find all resources with a `kube_deployment` tag value starting with `stats-`
+- `pod_name:*canary`: Find all resources with a `pod_name` value ending in `canary`
+- `label#release:*`: Find all resources with a `release` label, regardless of its value
+- `-label#*.datadoghq.com/*`: Find resources that do not have any Datadog-scoped labels.
+- `kube_*:*stats*canary`: Find resource with a related-resource tags (`kube_*`), with with  `stats` in the middle of the value, also ending with `canary`.
+
+### Extracted Tags
+
+In addition to the tags you have [configured](https://docs.datadoghq.com/getting_started/tagging/assigning_tags/?tab=containerizedenvironments) within your Datadog agent, Datadog injects generated tags based on resource attributes that can help your searching and grouping needs. These tags are added to resources conditionally, when they are relevant.
```

Comment:
```suggestion
In addition to the tags you have [configured][20] within your Datadog agent, Datadog injects generated tags based on resource attributes that can help your searching and grouping needs. These tags are added to resources conditionally, when they are relevant.
```
Can you change this link to the format noted here: https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1182958054/Markdown#Links

So for example it would be : 

`[configured][20]` 

then at the bottom of the markdown file:
[20]: /getting_started/tagging/assigning_tags/?tab=containerizedenvironments

(or a number not yet in use).